featuretests/agent_caasoperator_test.go

```diff
@@ -59,19 +60,19 @@ func (s *CAASOperatorSuite) newBufferedLogWriter() *logsender.BufferedLogWriter
 	return logger
 }
 
-func waitForApplicationActive(c *gc.C, agentDir string) {
+func waitForApplicationActive(c *gc.C, dataDir, applicationName string) {
 	timeout := time.After(coretesting.LongWait)
-
 	for {
 		select {
 		case <-timeout:
 			c.Fatalf("no activity detected")
 		case <-time.After(coretesting.ShortWait):
-			link := filepath.Join(agentDir, commands.CommandNames()[0])
+			agentBinaryDir := agenttools.ToolsDir(dataDir, "application-"+applicationName)
```

Comment:
The jujud operator binary doesn't live in a directory named after the application - it is baked into the operator docker image in /usr/bin or somewhere. That's why manifold config had an AgentDir config attribute and was set to os.Args[0]

Reply:
I see. Updated to look in /var/lib/juju/tools, and added a comment on Config.DataDir.